products/storage/terraform.yaml

```diff
@@ -14,7 +14,26 @@
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
   Bucket: !ruby/object:Overrides::Terraform::ResourceOverride
-    exclude: true
+    exclude_resource: true
+    import_format: ["{{name}}"]
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      allowed_iam_role: 'roles/storage.objectViewer'
+      admin_iam_role: 'roles/storage.admin'
+      parent_resource_attribute: 'bucket'
+      base_url: 'b/{{name}}'
+      import_format: ['b/{{name}}', '{{name}}']
+      iam_conditions_request_type: :QUERY_PARAM
+      fetch_iam_policy_method: 'iam'
+      set_iam_policy_method: 'iam'
+      set_iam_policy_verb: :PUT
+      wrapped_policy_obj: false
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "storage_bucket_basic"
+        primary_resource_id: "default"
+        vars:
+          name: "my-bucket"
+        primary_resource_name: "fmt.Sprintf(\"my-bucket%s\", context[\"random_suffix\"])"
```

Comment:
I don't see this used in the example template at all, is it still needed?

Reply:
Used to generate the ImportId in generated tests:
https://github.com/GoogleCloudPlatform/magic-modules/blob/75c7575ab41e03b21eb7c3f2e2464cf7d6800c77/templates/terraform/examples/base_configs/iam_test_file.go.erb#L66